src/calculateRentalCost.js

```diff
@@ -4,7 +4,23 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const LONG_TERM = 7;
+  const LONG_TERM_DISCOUNT = 50;
+  const SHORT_TERM = 3;
+  const SHORT_TERM_DISCOUNT = 20;
+  const PRICE_IN_DAY = 40;
+
+  let price = days * PRICE_IN_DAY;
```

Comment:
Consider using `const` instead of `let` for the `price` variable since it's not being reassigned but only modified within the conditional blocks.